addons/mx2a/stringtable.xml

```diff
@@ -0,0 +1,27 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project name="ACE">
+    <Package name="MX2A">
+        <Key ID="STR_ACE_MX2A_DisplayName">
+            <English>MX-2A</English>
+            <German>MX-2A</German>
+            <Spanish>MX-2A</Spanish>
+            <Czech>MX-2A</Czech>
+            <Russian>MX-2A</Russian>
+            <Italian>MX-2A</Italian>
+            <Polish>MX-2A</Polish>
+            <French>MX-2A</French>
+            <Hungarian>MX-2A</Hungarian>
+        </Key>
+        <Key ID="STR_ACE_MX2A_Description">
+            <English>Thermal Scope</English>
+            <German>Thermal Scope</German>
```

Comment:
The non-translated entries should be deleted (Spanish in particular, but afaik also Italian and French)